docker-compose-py3.yml

```diff
@@ -84,28 +85,46 @@ services:
       - LOG_FORMAT=plain
       - LOG_LEVEL=WARNING
 
-
+  # This container exists solely to have something automatically rebuilding the UI
+  # in the background during local development. Although it runs a web server, we
+  # don't use it. Instead, the nginx one below is used as our entry point to more
+  # closely mimic production, and support SSL.
   balrogui-py3:
     image: node:0.10
     depends_on:
       balrogadmin-py3:
         condition: service_healthy
-    ports:
-      - "18080:18080"
-    links:
-      - balrogadmin-py3
+    expose:
+      - 16060
```

Comment:
Is this `balrogui` still exposed in case it's useful for debugging nginx ?

Reply:
Hm, I can't remember why I did this now (this part was written last year). I suspect it was probably to help me verify some part of the nginx setup - but debugging is definitely enabled by it too. Now that you point it though, it makes me wonder if it's confusing to have both exposed...maybe we should just remove it, and let it stick to background rebuilding?